docs/azure/azure-explanation/canonical-offerings.rst

```diff
@@ -1,24 +1,60 @@
+Ubuntu images
+-------------
 
-**Customized kernel** - The ``linux-azure`` kernel enables accelerated networking for the InfiniBand capable instances, as well as consistent support for the Single Root I/O Virtualization (SR-IOV) on the present hardware, enabling network traffic to bypass the virtualization stack and achieve almost native performance. It comes with FPGA support out of the box, taking advantage of project catapult to provide performance without the cost and complexity of a custom ASIC.
+Canonical produces a wide variety of Ubuntu images to support numerous features found on Azure.
 
-Some other offerings include:
+* `Server images`_ - These are general-purpose, customized Azure Virtual Machines based on the ``linux-azure`` kernel. These images can also be enhanced with `Ubuntu Pro on LTS`_.
 
-1. Ubuntu server and Ubuntu Pro for all supported LTS versions across all Azure architectures, including ARM
-#. Minimal Ubuntu images for Independent Software Vendors and others who want to build on Azure
-#. Ubuntu Pro FIPS images that allow US government Ubuntu users to easily move their workloads to Azure
-#. Pre-hardened (CIS) Ubuntu minimal images, for those who want best security practices for Ubuntu out of the box
-#. The only commercial Linux launch partner for Azure's Confidential Computing platform (both AMD SEV-SNP and TDX)
-#. SQL Server on Ubuntu Pro images that include specific SQL-related OS optimizations and joint support from Canonical and Microsoft
-#. Anbox on Azure, that allows users to run Android apps on Azure at scale
-#. Collaboration with Azure's AKS team to support the Azure Kubernetes worker node image, as these worker nodes nearly always run Ubuntu
+* `Minimal images`_ - These images are designed for automated deployment at scale with a reduced default package set. Things like interactive usage tools are omitted. They are much smaller, boot faster, and require fewer security updates over time due to the fewer installed packages. These images can also be enhanced with `Ubuntu Pro on LTS`_.
+
+* `CVM images`_ - These Confidential VM images provide enhanced security features designed to protect data at rest, in use, and during boot. CVM images are Intended for use with Azure's confidential computing capabilities using hardware-enabled security features. These images can also be enhanced with `Ubuntu Pro on LTS`_.
+
+* GB200-Compatible Server images - These images are optimized for AI when enabled with `NVIDIA GB200 Grace Blackwell hardware`_. They are built with a custom linux-azure-nvidia kernel and a pre-configured RDMA/InfiniBand user-space. These images can also be enhanced with `Ubuntu Pro on LTS`_.
```

Comment:
Instead of "can also be enhanced with" it might be better to say something like "also available with Pro-enabled". The distinction I'm trying to convey is that we have existing images whereas enhancing implies the user might have to do this themselves.